### DISCUSSION TITLE
How to handle (generate & process) complex form?

### DISCUSSION THREAD

**Discussion by MagePsycho** [symfony/symfony#43962] [Q&A]
I have a Form that has multiple fieldsets and some of the fieldsets have an option to generate more fields (with an add-more button).
And these fields are not tied with the entity, they are just used to generate a file.

So how would you handle such form (to generate the UI with ACL & to process the submission, of course with validation)?

**Comment by yceruto** [comment]
I think the trick is explained in the documentation:
* https://symfony.com/doc/current/form/dynamic_form_modification.html
* https://symfony.com/doc/current/form/validation_group_service_resolver.html

**Reply by MagePsycho** [reply]
[UPDATED] Also, would like to have ACL connected with each field.
